Gordon360/Services/AcademicCheckInService.cs

```diff
@@ -0,0 +1,175 @@
+using Gordon360.Exceptions.CustomExceptions;
+using Gordon360.Models;
+using Gordon360.Models.ViewModels;
+using Gordon360.Repositories;
+using Gordon360.Services.ComplexQueries;
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace Gordon360.Services
+{
+    /// <summary>
+    /// Service Class that facilitates data transactions between the AcademicCheckInController and the CheckIn database model.
+    /// </summary>
+	/// 
+    public class AcademicCheckInService : IAcademicCheckInService
+    {
+        private IUnitOfWork _unitOfWork;
+        public AcademicCheckInService(IUnitOfWork unitOfWork)
+        {
+            _unitOfWork = unitOfWork;
+        }
+
+
+        /// <summary> Stores the emergency contact information of a particular user </summary>
+        /// <param name="data"> The object that stores the contact info </param>
+        /// <param name="id"> The students id number</param>
+        /// <returns> The stored data </returns>
+        public EmergencyContact PutEmergencyContact(EmergencyContact data, string id)
+        {
+            var studentIDParam = new SqlParameter("@StudentID", Int32.Parse(id));
+            var contactIDParam = new SqlParameter("@ContactNum", data.SEQ_NUM);
+            var contactLastNameParam = new SqlParameter("@ContactLastName", data.lastname);
+            var contactFirstNameParam = new SqlParameter("@ContactFirstName", data.firstname);
+            var contactHomePhoneParam = new SqlParameter("@ContactHomePhone", FormatNumber(data.HomePhone));
+            var contactMobilePhoneParam = new SqlParameter("@ContactMobilePhone", FormatNumber(data.MobilePhone));
+            var contactRelationshipParam = new SqlParameter("@ContactRelationship", data.relationship);
+            var notesParam = new SqlParameter("@Notes", GetNotesValue(data.MobilePhone, data.HomePhone));
+            var usernameParam = new SqlParameter("@Username", "360Web (" + data.lastname + ", " + data.firstname + ")");
+            var jobNameParam = new SqlParameter("@JobName", "Enrollment-Checkin");
+
+            // Run stored procedure
+            var result = RawSqlQuery<EmergencyContactViewModel>.query("UPDATE_EMRGCONTACT @StudentID, @ContactNum, @ContactLastName, @ContactFirstName, @ContactHomePhone, @ContactMobilePhone, @ContactRelationship, @Notes, @Username, @JobName", studentIDParam, contactIDParam, contactLastNameParam, contactFirstNameParam, contactHomePhoneParam, contactMobilePhoneParam, contactRelationshipParam, notesParam, usernameParam, jobNameParam);
+            if (result == null)
+            {
+                throw new ResourceNotFoundException() { ExceptionMessage = "The data was not found." };
+            }
+            return data;
+        }
+
+        private String GetNotesValue(String MobilePhone, String HomePhone)
```

Comment:
CreateNotesValue might be clearer.  Need comment.  Explain that  notes field is generated here, not added by a person.